packages/paste-website/src/components/shortcodes/component-changelog/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import Parser from 'html-react-parser';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Box} from '@twilio-paste/box';
+
+interface ComponentChangelogProps {
+  data?: {
+    childMarkdownRemark: {
+      html: string;
+      rawMarkdownBody: string;
+    };
+  };
+}
+
+export const StyledChangelogContent = styled(Box)`
+  h1 {
```

Comment:
These bugs me. Is there no way we can hook into the markdown and provide our own components to replace the HTML elements as we do in the mdx provider?

Reply:
Completely agree on this one. I tried wrapping the content with the mdx provider, but that didn't have any affect. Anything else you're aware of that I can dig into for this?

Reply:
I'll have a look tomorrow to see if I can find anything

Reply:
🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉